src/Microsoft.DotNet.XHarness.iOS.Shared/AppBundleInformation.cs

```diff
@@ -35,5 +35,5 @@ public class AppBundleInformation
     }
 
     public static AppBundleInformation FromBundleId(string bundleIdentifier) =>
-        new(bundleIdentifier, bundleIdentifier, string.Empty, string.Empty, false);
+        new(bundleIdentifier, bundleIdentifier, string.Empty, string.Empty, false, bundleExecutable: bundleIdentifier);
```

Comment:
Why are we setting properties like appName to be the same string as bundleIdentifier?

Reply:
Good question. I will add a comment here explaining that